cordova-lib/spec-cordova/platform.spec.js

```diff
@@ -354,22 +359,94 @@ describe('add function', function () {
         });
     });
 
-    it('fails if there is an error while getting the platform details', function (done) {
+    it('tries to git-clone if there is an error while getting the platform details as specified from the command line', function (done) {
 
-        var targets = ['C:\\Projects\\cordova-projects\\cordova-android'];
+        var targets = ['https://github.com/apache/cordova-android'],
+	    targetPlatform = 'android',
+	    libDir = 'C:\\Projects\\cordova-projects\\cordova-android-clone-directory';
+
+	var cloneGitRepoMock = jasmine.createSpy().andReturn(Q({
+	    platform: targetPlatform,
+	    libDir: libDir
+	}));
+	platform.__set__('cloneGitRepo', cloneGitRepoMock);
```

Comment:
@jsoref , why do I still need a PR to update https://github.com/apache/cordova-cli/blob/38dd67242841dd2ec254f7f9f402f495f7a28f5b/doc/platform.txt#L20 ?
It seems like the file already refers to <url>s